tools/run_tests/xds_k8s_test_driver/framework/xds_k8s_testcase.py

```diff
@@ -185,6 +187,12 @@ def assertAllBackendsReceivedRpcs(self, lb_stats):
 
 class RegularXdsKubernetesTestCase(XdsKubernetesTestCase):
 
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        if cls.server_maintenance_port is None:
+            cls.server_maintenance_port = server_app.KubernetesServerRunner.DEFAULT_MAINTENANCE_PORT
```

Comment:
I'm a bit surprised yapf isn't complaining about it
![image](https://user-images.githubusercontent.com/672669/118730166-2df04380-b805-11eb-8f43-4caced94cd50.png)

Reply:
Yeah... I think today only PyLint complains about line too long (>100), YAPF just formats it in best-effort. Manually folded this line.